src/jshint.js

```diff
@@ -4417,14 +4424,10 @@ var JSHINT = (function() {
 
       expression(10);
 
-      if (state.tokens.next.id === "class") {
```

Comment:
to explain this change, this was a typo - `state.tokens.next` is at this point the thing after an export. This made the following code fail
```
/* jshint undef: true */
export default class A{}
class B{}
A();
```